client/scss/utilities/_root-scope-classes.scss

```diff
@@ -317,6 +325,25 @@
   }
 }
 
+.promo-link {
+  display: block;
+  color: color('black');
+
+  &:hover .promo-link__title,
+  &:focus .promo-link__title {
+    color: color('mint');
+  }
+
+  @supports (display: grid) {
+    display: flex;
+  }
+
+}
```

Comment:
Wonder if these could be made more generic, but we don't really use them else where, so this is good.